ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,106 @@
+    *    This flag should be set alongside the `ENABLE_AZURE` flag.
+    *    If the `IMPORT_AZURE_SDK` flag is not specified, the compiler will assume a system installation of the SDK which will be implemented at a later stage.
+
+### Letting CMake manage the SDK dependency as an external project
+
+This method configures CMake to download, compile, and install the Azure SDK while building the Azure extension.
+
+```bash
+# Create a new directory to run your build from
+mkdir build && cd build
+
+# Configure and run cmake with Ninja
+cmake -DENABLE_PYTHON=1 -DHAVE_UNITTEST=1 -DIMPORT_AZURE_SDK=external -DENABLE_AZURE=1 -G Ninja ../.
+ninja
+```
+
+* The compiler flag `IMPORT_AZURE_SDK` must be set to `external` for this build method.
+* `ENABLE_AZURE` defaults to looking for a local version, the `IMPORT_AZURE_SDK` setting will override that default.
+## 3. Development
+In order to run this extension after building, the developer must have an Azure connection string locally to a container with the right permissions. The connection string must be stored in an environmental variable called `AZURE_STORAGE_CONNECTION_STRING`. To store your connection string into an environmental variable type `export AZURE_STORAGE_CONNECTION_STRING="your Azure connection string"` into your terminal.
+## 4. Testing
+
+Before running the tests set the `LD_LIBRARY_PATH` to tell the dynamic link loader where to look for the dynamic shared libraries that we made earlier.
+```bash
+export LD_LIBRARY_PATH=$(pwd)/azure-sdk-cpp/install/lib:$LD_LIBRARY_PATH
+```
+
+### To run the tiered python tests for Azure:
+
+```bash
+# This will run all the tests in test_tiered19.py on the Azure storage source. The following command will run the tests from the build directory that was made earlier.
+cd build
+env WT_BUILDDIR=$(pwd) python3 ../test/suite/run.py -j 10 -v 4 test_tiered19
+```
+
+### To run the C unit tests for Azure:
+
+```bash
+# Once WiredTiger has been built with the Azure Extension, run the tests from the build directory
+cd build
+ext/storage_sources/azure_store/test/run_azure_unit_tests
+```
+
+To add any additional unit testing, add to the file `test_azure_connection.cpp`, alternatively if you
+wish to add a new test file, add it to the `SOURCES` list in `create_test_executable()`
+(in `azure_store/test/CMakeLists.txt`).
+
+## 5. Evergreen Testing
+Currently the Evergreen testing runs both `test_tiered19.py` and the unit tests in `test_azure_connection.cpp`. Should a developer wish to additional tests to the extension, they would first have to write the tests before adding it as a task to the evergreen.yml file.
+
+Evergreen has hidden the connection string for Azure so it's necessary for the developer to store this in the environmental variable in Step 3.
+
+When creating a new task, a developer should note that the CMake binary should be set to the MongoDB V4 toolchain due to the CMake 3.13 minimum requirement.
```

Comment:
Can you fix the line lengths a bit so it's still readable in a terminal/text editor rather than just the rendered Markdown?

Reply:
yup i've adjusted it in https://github.com/wiredtiger/wiredtiger/pull/8802/commits/7a0835b2f48fa7ee3afc558eac783f798980748f so it fits the 100 char limit let me know if there are any changes that need to be made